src/Runtime/OMTensor.inc

```diff
@@ -344,6 +346,76 @@ void *omTensorGetAllocatedPtr(const OMTensor *tensor) {
   return tensor->_allocatedPtr;
 }
 
+void omTensorPrint(const OMTensor *tensor) {
+  OM_DATA_TYPE dataType = omTensorGetDataType(tensor);
+  int64_t rank = omTensorGetRank(tensor);
+  const int64_t *shape = omTensorGetShape(tensor);
+  const int64_t *strides = omTensorGetStrides(tensor);
+  void *dataPtr = omTensorGetDataPtr(tensor);
+
+  printf("\trank = %lld\n", (long long)rank);
+  printf("\tdataType = %s\n", getDataTypeName(dataType));
+
+  if (rank == 1) {
```

Comment:
Why only rank == 1? I think we have the shape info for any rank.
Make sure that the output is in cvs format.

Reply:
Yes, Numpy for multiple dimension array.

Reply:
Fair question. It is a current arbitrary limitation because when I wrote this code I was debugging a case with a 1D tensor. I am thinking to extend it to support 2D tensors in this PR, and if we need a solution for any dimensions extend the code at that point?  Acceptable ?

Reply:
sure.

Reply:
Done, we can now print OMTensor up to rank <= 3.

Reply:
The data is printed out like this: 

```
Test 1D
        rank = 1
        dataType = int64_t
        numElems = 3
        strides: [1]
        data: ([1, 2, 3])
Test 2D
        rank = 2
        dataType = float
        numElems = 6
        strides: [2][1]
        data: ([[1, 2], [3, 4], [5, 6]])
Test 3D
        rank = 3
        dataType = float
        numElems = 6
        strides: [2][1][1]
        data: ([[[1], [2]], [[3], [4]], [[5], [6]]])

Reply:
nice, we can then just substitute `data:` for `np.array` and the "()" are already there (which is very very convenient when the arrays has 10k elements...)